c7n/resources/aws.py

```diff
@@ -138,7 +138,7 @@ class MetricsOutput(Metrics):
     permissions = ("cloudWatch:PutMetricData",)
     retry = staticmethod(utils.get_retry(('Throttling',)))
 
-    def __init__(self, ctx, config=None):
+    def __init__(self, ctx, config={}):
```

Comment:
mutables in default args are always suspect

Reply:
yup, reverted